libs/mini-rx-store/src/lib/models.ts

```diff
@@ -1,5 +1,11 @@
 import { Observable } from 'rxjs';
-import { StoreExtension } from '@mini-rx/common';
+import {
+    Action,
+    StoreExtension,
+    StateOrCallback,
+    MetaReducer,
+    ReducerDictionary,
+} from '@mini-rx/common';
 
 export type AppState = Record<string, any>;
```

Comment:
I think some models here can be cleaned up:

My first guess would be:

![image](https://github.com/spierala/mini-rx-store/assets/1272446/9f52444c-1192-428a-beaa-3d1f1c543fde)